spec/spec.md

```diff
@@ -0,0 +1,180 @@
+the project. The name of the license file should be the SPDX identifier of the
+license followed by a file extension (example: `GPL-3.0-or-later.txt`).
+
+<!-- FIXME: Rephrase "above recommendation". Maybe use paragraph numbers? -->
+
+A license file whose name does not conform to the above recommendation must
+contain, at the start of the file, one or more `Valid-License-Identifier` tags,
+followed by a `License-Text` tag, followed by the license text. Example:
+
+```
+Valid-License-Identifier: GPL-3.0-only
+Valid-License-Identifier: GPL-3.0-or-later
+License-Text:
+
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
+
+[...]
+```
+
+A project may contain a single license file outside of the `LICENSES/` directory
+in the file `COPYING`, `COPYRIGHT`, `LICENSE` or `LICENCE` in the root of the
+project.
+
+If a license does not exist in the SPDX License List, its SPDX identifier is
+`LicenseRef-[idstring]` as defined by the SPDX Specification.
+
+A project must not include license files for licenses under which none of the
+files in the project are licensed.
+
+Everything that applies to licenses in this section also applies to license
+exceptions, with the following exceptions:
+
+- Exception files must be placed in the `LICENSES/` directory in the root of the
+  project.
+
+- The tags `Valid-Exception-Identifier` and `Exception-Text` must be used
+  instead of `Valid-License-Identifier` and `License-Text`.
+
+<!-- Can we use LicenseRef for exceptions? -->
+
+<!-- TODO: Change "copyright and licensing information" to something shorter?
+Shorten it to CALI? -->
+
+## Copyright and licensing information
+
+Each file in the project must have copyright and licensing information
+associated with it, except the following files:
+
+- The license files.
+
+- The files belonging to the project's version control system (example:
+  `.git/`).
+
+- The files ignored by the version control system (example: `.gitignore`).
+
+- The files in the `.reuse/` directory in the root of the project.
+
+<!-- TODO: The files inside of a git submodule -->
+
+There are various ways to associate copyright and licensing information with a
+file.
+
+### Comment headers
+
+Each plain text file that can contain comments should contain comments at the
+top of the file (comment header) that declare that file's copyright and
+licensing information.
+
+If a file is not a plain text file or does not permit the inclusion of
+comments, the comment header that declares the file's copyright and licensing
+information should be in an adjacent file of the same name with the
+additional extension `.license` (example: `cat.jpg.license` if the original
+file is `cat.jpg`).
+
+The comment header must contain one or more `SPDX-Copyright` tags, and one or
```

Comment:
```suggestion
The comment header MUST contain one or more `SPDX-Copyright` tags, and one or
```

Reply:
FYI - The SPDX-Copyright tag is not yet included in the spec.  We are tracking an issue to add it here: https://github.com/spdx/spdx-spec/issues/122

Reply:
Correct, I'm aware. This spec won't be published until there is a consensus in SPDX about such an added tag. Thanks for the link to the issue, though!

Reply:
Fixed in 54b7d330d56ce1912f6d949f8745730aa20b1fc5, though still waiting on https://github.com/spdx/spdx-spec/issues/122